krcg_api/api.py

```diff
@@ -191,15 +191,17 @@ def random_deck():
 @base.route("/convert/<format>", methods=["POST"])
 def convert(format="json"):
     raw_data = flask.request.get_data()
-    if flask.request.json:
-        d = deck.Deck()
-        d.from_json(flask.request.json)
-    else:
+    if flask.request.content_type == "text/plain":
         try:
             text = io.StringIO(raw_data.decode("utf-8"))
             d = deck.Deck.from_txt(text)
         except UnicodeDecodeError:
             return "Failed to decode text/plain data in utf-8", 400
+
+    if flask.request.content_type == "application/json":
+        d = deck.Deck()
+        d.from_json(flask.request.json)
+
```

Comment:
maybe use if / elif / else: we don't want a 500 if the content_type was not properly set. I would argue that defaulting to any content type other than json to try the plain text treatment made more sense than this change.

Reply:
Plus this seems to have nothing to do with the PR intent, maybe simply remove?